### PR TITLE
Update README to reflect new astria-proto structure

### PR DESCRIPTION
**Pull request by joroshiba**

### CODE REVIEW

File: crates/astria-proto/README.md [hunk 1]
```diff
@@ -1,24 +1,18 @@
...
-* define protos in `sandwich-apis/proto/`
-* create `buf.gen.yaml` and `buf.yaml` in `sandwich-apis/`.
-* use `execution-apis/` as a reference for `buf.gen.yaml` and `buf.yaml`.
-* create `build.rs`, `touch sandwich-apis/build.rs`
-* copy over logic from `execution-apis/build.rs` for now, updating necessary paths
-* update `sandwich-apis/src/lib.rs` to include the files generated from `build.rs`. see `execution-apis/src/lib.rs`
-* add steps to `.github/workflows/pull-request.yml` and `.github/workflows/push.yml` to lint, build, and push the buf image from the new proto defs
+* Create a new folder under `proto/astria/{new_package}`
+* write protos in new folder using package name `astria.{new_package}`
+* update `src/lib.rs` to build a rust module from your new defintions
```
Comment (noot) [inline crates/astria-proto/README.md:15]: it's actually the build script (build.rs) that's doing the building, lib.rs just exports the rust artifacts of building the protos, so maybe reword this to `update `src/lib.rs` to include your new module`?

**Review by noot — APPROVED** [review]

**Review by steezeburger — APPROVED** [review]
